Resources/ServerInfo/Guidebook/Science/Robotics.xml

```diff
@@ -0,0 +1,51 @@
+
+  ## Exosuit Fabricator
+  <Box>
+<GuideEntityEmbed Entity="ExosuitFabricator" Caption="Exosuit Fabricator"/>
+</Box>
+  The [color=#a4885c]Exosuit Fabricator[/color] is where you'll be printing out most of the parts for your creations, such as:
+  - Bot limbs
+  - Mech parts and equipment
+  - Proximity sensors
+
+  Operating the exosuit fabricator is the same as every other lathe on the station; simply click the fabricator to open the UI, then click the icon for whatever part you want to print.
+  - Are there no recipes? click the [color=#a4885c]sync[/color] button on the UI to check for any new blueprints. If, even then, nothing shows up, you've likely not researched the relevant technology yet!
+  - If a recipe appears grayed-out, it's likely due to a lack of materials. To insert new materials (either [color=#99A3A4]steel[/color], [color=#ECF0F1]plastic[/color], or [color=#AED6F1]glass[/color]), simply click the fabricator while holding said material.
+
+  Every station starts with a full suite of robotics equipment by default, but you can make another fabricator if [color=#a4885c]Robotics[/color] technology is available, which will allow you to make the respective circuit board.
+
+  ## Bots
+  <Box>
+<GuideEntityEmbed Entity="MobCleanBot" Caption="Cleanbot"/>
+</Box>
+  [color=#a4885c]Bots[/color] are small, simple automata which can perform simple tasks, such as cleaning. They don't use power cells, but are quite fragile. Making these will require [color=#a4885c]Robotics[/color] technology in order to fabricate the robotic arms and proximity sensors necessary to assemble them.
+
+  To make a bot, first open the [color=#a4885c]Build Menu[/color] (bound to the 'G' key by default). Open the "Robots" tab, and you'll have the whole list of Bot recipes.
+
+  In order to actually assemble a bot from this menu, you must have the parts outlined by the recipe somewhere in your inventory. They can be in your hands, pockets, or even your backpack. To build the bot, just click [color=#a4885c]Build[/color], and your newly created silicon servant should appear at your feet!
+
+  ## Mechs
+  <Box>
+<GuideEntityEmbed Entity="MechRipley" Caption="Ripley APLU"/>
+</Box>
+  [color=#a4885c]Mechs[/color] are giant behemoths of metal designed to carry out a variety of functions respective to their model. They are piloted by players and require power cells to function.
+
+  Each mech requires its own unique set of limbs and parts to create. These can be fabricated at the Exosuit Fabricator, the same as any other part, save for the required [color=#a4885c]circuit boards[/color], which are made at a [color=#a4885c]Circuit Board Imprinter[/color], which can also be found in the science department.
+
+  All mechs require assembly before usage; after slotting every limb into the chassis, [color=#a4885c]examine[/color] it to see how to proceed with construction.
```

Comment:
you don't mention here that you need to place the construction ghost from the menu, really most construction mentions should be at the start, mentioning the construction guide or something

Reply:
mechs don't use the construction menu...

Reply:
yeah they don't